src/components/nodes/pages/sup-book/digital-project-page/digital-project-page.tsx

```diff
@@ -42,214 +42,225 @@ const DigitalProjectPage = async ({node, ...props}: Props) => {
+            </div>
+            <div className="mx-auto max-w-5xl">
+              {node.supBookDescription?.processed && (
+                <TabPanel>
+                  <Wysiwyg html={node.supBookDescription?.processed} />
+                </TabPanel>
+              )}
+              {node.supBookReviews && (
+                <TabPanel>
+                  <Wysiwyg html={node.supBookReviews.processed} />
+                </TabPanel>
+              )}
+              {node.supBookAuthorInfo && (
+                <TabPanel>
+                  <Wysiwyg html={node.supBookAuthorInfo.processed} />
+                </TabPanel>
+              )}
             </div>
+          </Tabs>
+        )}
 
-            {hasExcerptAndMore && (
-              <Link
-                href={`${node.path}/excerpts`}
-                className="group rs-py-1 rs-mt-2 rs-px-2 mx-auto flex w-fit items-center justify-center gap-5 border-2 border-press-sand font-normal text-stone-dark no-underline hocus:border-cardinal-red hocus:bg-cardinal-red hocus:text-white hocus:underline md:mt-0"
-              >
-                <span>Excerpts + more</span>
-                <DocumentDuplicateIcon width={28} className="text-stone group-hocus:text-white" />
-              </Link>
-            )}
-          </div>
-        </div>
-      </>
-
-      {(node.supBookDescription?.processed || node.supBookReviews || node.supBookAuthorInfo) && (
-        <Tabs className="mb-20 border-b border-fog pb-20">
-          <div className="mb-20 border-b border-fog">
-            <TabsList className="mx-auto max-w-5xl">
-              {node.supBookDescription?.processed && <Tab className="p-10">Description</Tab>}
-              {node.supBookReviews && <Tab className="p-10">Reviews</Tab>}
-              {node.supBookAuthorInfo && <Tab className="p-10">About the Author</Tab>}
-            </TabsList>
-          </div>
+        {node.supBookSubjects && (
           <div className="mx-auto max-w-5xl">
-            {node.supBookDescription?.processed && (
-              <TabPanel>
-                <Wysiwyg html={node.supBookDescription?.processed} />
-              </TabPanel>
-            )}
-            {node.supBookReviews && (
-              <TabPanel>
-                <Wysiwyg html={node.supBookReviews.processed} />
-              </TabPanel>
-            )}
-            {node.supBookAuthorInfo && (
-              <TabPanel>
-                <Wysiwyg html={node.supBookAuthorInfo.processed} />
-              </TabPanel>
-            )}
+            <H2 className="type-0 font-bold">Related Subjects</H2>
+            <ul className="list-unstyled flex flex-col md:flex-row md:flex-wrap">
+              {node.supBookSubjects.map(subject => {
+                const linkParamsString = createLinkParams(subject)
+                return (
+                  <li key={subject.id} className="min-w-fit flex-1">
+                    <a
+                      href={`/search?${linkParamsString}`}
```

Comment:
Optionally, you could use call `createLinkParams` directly in the href prop. Then you don't need to have a `return` and stuff.